app/views/SetUsernameView.js

```diff
@@ -98,6 +98,8 @@ class SetUsernameView extends React.Component {
 			log(e);
 		}
 		this.setState({ saving: false });
+		const { navigation } = this.props;
```

Comment:
Same of comment above.

Reply:
I have made the changes, now you can verify that sir .